frontend/src/Lighting.jsx

```diff
@@ -32,40 +32,24 @@ function Lighting() {
     if (!lightOn) {
       setLightIntensity(0);
       setLightColor(0);
-      setDetermineColor('None');
       setStringColor('#000000');
-    } else if (lightColor <= 0.5) {
-      setDetermineColor('Yellow');
-      setStringColor('#ffd700');
-    } else if (lightColor < 1.0) {
-      setDetermineColor('Blue');
-      setStringColor('#00bfff');
-    } else if (lightColor < 1.5) {
-      setDetermineColor('Pink');
-      setStringColor('#ff69b4');
-    } else if (lightColor < 2.0) {
-      setDetermineColor('Red');
-      setStringColor('#b22222');
-    } else if (lightColor < 2.5) {
-      setDetermineColor('Green');
-      setStringColor('#008000');
     } else {
-      setDetermineColor('Purple');
-      setStringColor('#9400d3');
+      setStringColor(`hsl(${lightColor}, ${lightIntensity}%, 50%)`);
```

Comment:
If the stringColor contains the information of lightColor and lightIntensity, then do we need to store lightColor and lightIntensity into the AsyncStorage?

Reply:
stringColor stores the HSL value, while lightColor is the hue information. Maybe I could rename that stuff to make it clearer? because lightColor has to be incremented/decremented by the slider

Reply:
I guess maybe I don't understand exactly what HSL is or how it works. I'm not saying to remove lightColor, because we use it in the slider as you said, but if the HSL value contains that information, we don't need to redundantly store lightColor into AsyncStorage. We can just load stringColor from AsyncStorage onFocus, and then parse it for the lightColor and setLightColor then.

Reply:
Yeah that makes sense. It's redundant to store stringColor because it depends on lightColor and lightIntensity, so I'll remove that from AsyncStorage instead of lightColor.

Reply:
Well if stringColor holds both lightColor and lightIntensity, we can just store that instead of both lightColor and lightIntensity. Better to store one object than two.

Reply:
I'm just gonna go ahead and resolve this because we'll be using AWS Amplify DataStore to hold this data anyways instead of AsyncStorage (see #97) so I'll finalize our data models then

Reply:
Actually @vinaynagappala I don't think it really makes sense to store the lighting data in the backend. This data doesn't really need to be stored in the cloud because it's pretty local. We should finalize what data we really need to store on the backend and what data should stay local.

Reply:
ok cool, I'll rework the storage as you suggested. should we discuss what data is backend vs. local at this weekend's meeting?

Reply:
Yeah that's probably a good idea